### PR TITLE
Fixes a race condition when Livewire uploads is used on busy websites

### PR DESCRIPTION
**Pull request by bzarzuela**
1️⃣ Is this something that is wanted/needed? Did you create an issue / discussion about it first?
Yes, issue #2491 2491

2️⃣ Does it contain multiple, unrelated changes? Please separate the PRs out.
No

3️⃣ Does it include tests, if possible? (Not a deal-breaker, just a nice-to-have)
No, unfortunately I don't know how to test race conditions

4️⃣ Please include a thorough description of the improvement and reasons why it's useful.
On very busy websites, the cleanup code can run in multiple threads causing part of the output of allFiles() to have already been deleted by another thread.

5️⃣ Thanks for contributing! 🙌

### CODE REVIEW

File: src/WithFileUploads.php [hunk 1]
```diff
@@ -96,9 +96,11 @@ protected function cleanupOldUploads()
         $storage = FileUploadConfiguration::storage();
 
         foreach ($storage->allFiles(FileUploadConfiguration::path()) as $filePathname) {
-            $yesterdaysStamp = now()->subDay()->timestamp;
-            if ($yesterdaysStamp > $storage->lastModified($filePathname)) {
-                $storage->delete($filePathname);
+            if ($storage->exists($filePathname)) {
```
Comment (nuernbergerA) [inline src/WithFileUploads.php:99]: you could reduce cyclomatic complexity by continue early
```php
if (! $storage->exists($filePathname)) {
    continue;
}

$yesterdaysStamp = now()->subDay()->timestamp;
if ($yesterdaysStamp > $storage->lastModified($filePathname)) {
    $storage->delete($filePathname);
}

**Review by nuernbergerA — CHANGES_REQUESTED** [review]

File: src/WithFileUploads.php (hunk 1 above)
Comment (bzarzuela) [reply src/WithFileUploads.php:99]: You're right, this is a better solution. How do I pull in your suggestion in a way that is properly attributed to you? I'm not that familiar with this feature.

Comment (nuernbergerA) [reply src/WithFileUploads.php:99]: Just commit it I have enough contributions.

And "my way" is just formating your code different 👍

**Review by nuernbergerA — APPROVED** [review]

**Comment by calebporzio** [comment]
Good catch. Thanks! I understand the difficulty of testing this sort of thing, so I'm good with what you have.

**Pull request merged**